src/pages/_components/FullSpeedAhead.astro

```diff
@@ -25,7 +25,10 @@ import LandingSectionText from "./LandingSectionText.astro"
 				<CardFooter>
 					<p class="text-center">
 						Based on <a href="https://httparchive.org/" class="link-underline">HTTP Archive</a> real-world
-						performance data (Lighthouse, P90) • Full report coming soon (March 7th)
+						performance data (Lighthouse, P90) • <a
+							href="/blog/web-framework-performance-report/"
+							class="link-underline">Read the full report</a
```

Comment:
```suggestion
							class="link-underline">Read the 2023 performance report</a
```

Reply:
I think "full report" works here, any reasoning behind the change?